app/src/main/java/com/keylesspalace/tusky/fragment/TimelineFragment.java

```diff
@@ -561,6 +568,48 @@ public void onLoadMore(int position) {
         }
     }
 
+    /**
+     * Called when the status {@link android.widget.ToggleButton} responsible for collapsing long
+     * status content is interacted with.
+     *
+     * @param isCollapsed Whether the status content is shown in a collapsed state or fully.
+     * @param position    The position of the status in the list.
+     */
+    @Override
+    public void onContentCollapsedChange(boolean isCollapsed, int position) {
+        if(position < 0 || position >= statuses.size()) {
+            Log.e(TAG, String.format("Tried to access out of bounds status position: %d of %d", position, statuses.size() - 1));
+            return;
+        }
+
+        StatusViewData status = statuses.getPairedItem(position);
+        if(!(status instanceof StatusViewData.Concrete)) {
```

Comment:
merge this check into check above, there's no need to be so precise in errors in this case